CONTRIBUTING.md

```diff
@@ -84,14 +84,12 @@ be sure to check out the awesome [article about forking](https://help.github.com
 on the GitHub Help website &mdash; it will get you started quickly.
 
 You should always write each batch of changes (feature, bugfix, etc.) in **its own topic branch**.
-Please do not commit to the `main` branch, or your unrelated changes will go into the same pull request.
+Please do not commit to the `main` branch of your fork else your unrelated changes will go into the same pull request.
```

Comment:
This sounds a little confusing to me without punctuation, perhaps something like this is clearer:
```suggestion
Please do not commit to the `main` branch of your fork — otherwise your unrelated changes will go into the same pull request.
```